scripts/localstack-setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
maybe update the script name to better reflect that its unique to testing

Reply:
[74f2414](https://github.com/base-org/pessimism/pull/200/commits/74f241413764e3b9caf17a604ffe27f2bc28480c)